lib/blockscore/connection.rb

```diff
@@ -1,66 +1,57 @@
 module BlockScore
   module Connection
     MAJOR_VERSION = BlockScore::VERSION.split('.').first
-    REPO          = 'https://github.com/BlockScore/blockscore-ruby'.freeze
-    API_VERSION   = "version=#{MAJOR_VERSION}".freeze
+    REPO = 'https://github.com/BlockScore/blockscore-ruby'.freeze
+    API_VERSION = "version=#{MAJOR_VERSION}".freeze
     ACCEPT_HEADER = "application/vnd.blockscore+json;#{API_VERSION}".freeze
-    USER_AGENT    = "blockscore-ruby/#{BlockScore::VERSION} (#{REPO})".freeze
-    CONTENT_TYPE  = 'application/json'.freeze
-    API_URL       = URI.parse('https://api.blockscore.com').freeze
+    USER_AGENT = "blockscore-ruby/#{BlockScore::VERSION} (#{REPO})".freeze
+    CONTENT_TYPE = 'application/json'.freeze
+    API_URL = URI.parse('https://api.blockscore.com').freeze
+    HEADERS = {
+      'Accept' => ACCEPT_HEADER,
+      'User-Agent' => USER_AGENT,
+      'Content-Type' => CONTENT_TYPE
```

Comment:
I'm not sure. I would hope so, but I can't recall looking for it since it's a habit I've carried over from Perl.